### PR TITLE
Fixed build issues

### PR DESCRIPTION
**Pull request by linuskar**
Fixed issues that happened with building sprint 4 version of project. Can see the new build in latest release. For example changed not using AssetDatabase for wiki as it is part of Unity Editor namespace, which is not accessible for the built game.

### CODE REVIEW

File: DAT257AgileProject/Assets/Scripts/UI/Menus/WikiContainer.cs [hunk 1]
```diff
@@ -15,7 +15,17 @@ public class WikiContainer : MonoBehaviour
...
+        string[] folderPaths = {
+            "Assets/Resources/ScriptableObjects/TrashFacts/CarTire",
+            "Assets/Resources/ScriptableObjects/TrashFacts/CigaretteButt",
+            "Assets/Resources/ScriptableObjects/TrashFacts/ElectricScooter",
+            "Assets/Resources/ScriptableObjects/TrashFacts/GlassBottle",
+            "Assets/Resources/ScriptableObjects/TrashFacts/PETBottle",
+            "Assets/Resources/ScriptableObjects/TrashFacts/Plastic",
+            "Assets/Resources/ScriptableObjects/TrashFacts/PlasticSpoon",
+            "Assets/Resources/ScriptableObjects/TrashFacts/TrashBag"
+        };
```
Comment (Caramajau) [inline DAT257AgileProject/Assets/Scripts/UI/Menus/WikiContainer.cs:27]: So this was necessary?

Comment (Caramajau) [reply DAT257AgileProject/Assets/Scripts/UI/Menus/WikiContainer.cs:27]: Maybe possible to use normal System IO classes? https://stackoverflow.com/questions/56300890/alternative-to-assetdatabase-loadassetatpath-for-any-path

**Review by Caramajau — APPROVED** [review]
Left a comment, however if it is not possible to change, then it's fine.

File: DAT257AgileProject/Assets/Scripts/UI/Menus/WikiContainer.cs (hunk 1 above)
Comment (Oskmey) [reply DAT257AgileProject/Assets/Scripts/UI/Menus/WikiContainer.cs:27]: Could we use Resources.LoadAll? I think it is a part of engine namespace

**Review by Oskmey — APPROVED** [review]

File: DAT257AgileProject/Assets/Scripts/UI/Menus/WikiContainer.cs (hunk 1 above)
Comment (linuskar) [reply DAT257AgileProject/Assets/Scripts/UI/Menus/WikiContainer.cs:27]: > Could we use Resources.LoadAll? I think it is a part of engine namespace

Resources.LoadAll would load the objects not getting the folder paths

Comment (linuskar) [reply DAT257AgileProject/Assets/Scripts/UI/Menus/WikiContainer.cs]: > Maybe possible to use normal System IO classes? https://stackoverflow.com/questions/56300890/alternative-to-assetdatabase-loadassetatpath-for-any-path

Looks too much work to do for this small thing, some restructuring or reworking of the code.

**Pull request merged**